src/Raven.Server/Smuggler/Migration/Migrator_V2.cs

```diff
@@ -108,7 +115,7 @@ private async Task MigrateAttachments(string lastEtag)
                     var attachmentsArray = await GetAttachmentsList(lastEtag, transactionOperationContext);
                     if (attachmentsArray.Length == 0)
                     {
-                        var count = Parameters.Result.Documents.Attachments.ReadCount;
+                        var count = Parameters.Result.Documents.ReadCount;
```

Comment:
This looks bizarre? Why are we not using the Attachments here?

Reply:
It is the same number. Migrating legacy attachment creates a new document + attachment.

Reply:
we write the attachment as dummy doc therefore its proper to increase docs count.
https://github.com/ravendb/ravendb/blob/v4.1/src/Raven.Server/Smuggler/Migration/AbstractLegacyMigrator.cs#L104

Reply:
How this is displayed in the Studio? Shouldn't we increase both numbers?

Reply:
we increase the docs count here and the attachments count increased by DatabaseItemType.Attachments so in studio we see both docs and attachments
https://github.com/ravendb/ravendb/blob/v4.1/src/Raven.Server/Smuggler/Documents/DatabaseDestination.cs#L171